QUICK-START.md

```diff
@@ -78,11 +80,20 @@ release package.
 Populate the `Host`, `BootstrapUsername`, `BootstrapPassword`, and `BootstrapTenant` fields with the
 corresponding values for your Cumulocity tenant. The `Host` field should be populated with the
 MQTT broker's host for your Cumulocity instance, for example, `mqtt.eu-latest.cumulocity.com` or
-`mqtt.us-latest.cumulocity.com`. Do not include the scheme or port as they are automatically 
-selected to be https:// and 8883.
+`mqtt.us-latest.cumulocity.com`. Do not include the scheme or port as they are automatically selected to be https:// and 8883.
```

Comment:
Unnecessary change. Please revert back to the original (red) and amend commit. Let me know if you need help.